CHANGELOG.md

```diff
@@ -0,0 +1,15 @@
+# Changelog - chai-webdriverio
+
+## Unreleased Changes
+#### Breaking Changes:
+- Changed `there` and `visible` from methods to properties:
```

Comment:
I think we should leave ```there``` and ```visible``` as methods, at least until chai 4, because if you forget to initialise the plugin this will pass:

```
expect(selector).to.be.visible
```

Which is my main complaint against chai-3 property assertions, it's very easy to accidentally create assertion-less test cases